plugin/tests/unit/sdk_v2/plugin/test_test_plugin_v2_smoke.py

```
@@ -0,0 +1,378 @@
+        return self.config_data
+
+    async def get_own_base_config(self, timeout: float = 5.0):
+        return {"config": dict(self._base_config)}
+
+    async def get_own_profiles_state(self, timeout: float = 5.0):
+        return {"data": self._profiles_state}
+
+    async def get_own_profile_config(self, profile_name: str, timeout: float = 5.0):
+        return {"data": {"config": dict(self._profile_configs.get(profile_name, {}))}}
+
+    async def get_own_effective_config(self, profile_name: str | None = None, timeout: float = 5.0):
```

Ruff (0.15.5)

[warning] Unused method argument: `timeout`

(ARG002)